src/runtime/composables/useCldVideoUrl.ts

```diff
@@ -1,32 +1,32 @@
-import { useRuntimeConfig } from '#imports'
-import { constructCloudinaryUrl, ConstructUrlProps, ConfigOptions, AnalyticsOptions, VideoOptions } from '@cloudinary-util/url-loader'
-import nuxtPkg from 'nuxt/package.json';
+import { constructCloudinaryUrl, type ConstructUrlProps, type ConfigOptions, type AnalyticsOptions, type VideoOptions } from '@cloudinary-util/url-loader'
+import nuxtPkg from 'nuxt/package.json'
 import pkg from '../../../package.json'
```

Comment:
It may be better to have a virtual file which export only the info you need

Reply:
Sounds interesting. I will add it as a feature for next Release Candidate version :)